client/memconfig.go

```diff
@@ -0,0 +1,153 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	memConfigEndpoint = "/memstore/config"
+)
+
+// MemConfig ... contains properties that are used to configure the MemStore's behavior.
+// this is copied directly from /store/generated_key/memstore/memconfig. 
+// importing the struct isn't possible since it'd create cyclic dependency loop 
+// with core proxy's go.mod
+type MemConfig struct {
+	MaxBlobSizeBytes uint64
+	BlobExpiration   time.Duration
+	PutLatency time.Duration
```

Comment:
dang this is really unfortunate. What we could do is have core import from here instead? Kind of ugly though...

At some point we might need to refactor our stuff and create a `shared` workspace that has these kinds of structs that are shared between proxy and clients.